### DISCUSSION TITLE
<apex:repeat>

### DISCUSSION THREAD

**Discussion by stephenroden** [forcedotcom/sfdx-scanner#1445] [Q&A]
The scanner picks up <apex:repeat value="{!**varname**}" ...> as being a "Security-VfUnescapeEl" issue.  How are we supposed to escape varname so that it's not highlighted as an issue?

**Comment by johnbelosf** [comment]
@stephenroden apologies - I missed this discussion. Have you looked at: https://developer.salesforce.com/docs/atlas.en-us.pages.meta/pages/pages_security_tips_scontrols.htm

**Comment by stephen-carter-at-sf** [comment]
Closing older discussions now that we have gone GA with `code-analyzer`. See https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/get-started.html and if you still have questions, feel free to start a new discussion.
